Sources/SimpleSalesforceSDK/Models/SFAuth.swift

```diff
@@ -79,13 +78,19 @@ class SFAuth {
     let requestConfig = RequestConfig(url: url, params: params.data(using: .utf8))
     let request = URLRequestBuilder.request(with: requestConfig)
 
-    let (data, _) = try! await WebService.makeRequest(request)
-    let decoder = JSONDecoder()
-    decoder.dateDecodingStrategy = .secondsSince1970
-    let responseData = try! decoder.decode(RefreshTokenResponse.self, from: data)
+    do {
+      let (data, _) = try await WebService.makeRequest(request)
+      let decoder = JSONDecoder()
+      decoder.dateDecodingStrategy = .secondsSince1970
+      let responseData = try! decoder.decode(RefreshTokenResponse.self, from: data)
 
-    self.accessToken = responseData.accessToken
-    try! await self.interospectAccessToken(config: config)
+      self.accessToken = responseData.accessToken
+      try! await self.interospectAccessToken(config: config)
+    } catch {
+      self.reset()
+      print("Error while refreshing the access token...")
```

Comment:
Why aren’t we raising the error again after resetting the auth? How will the caller of the method know something went wrong?